### PR TITLE
added tooltip to EasyPill delete action

### PR DESCRIPTION
**Pull request by ManoloAlvarezForo**
- Added tooltip in addon EasyPill
- Added tooltip direction prop
- Unit test
- updated story for EasyPill

### CODE REVIEW

**Review by ftrinidad-cision — APPROVED** [review]

File: src/components/EasyPill/EasyPill.module.css [hunk 1]
```diff
@@ -2,3 +2,11 @@
   align-self: center;
   margin-right: var(--rvr-space-md);
 }
+
+.tooltipText {
+  height: 24px;
+  line-height: 24px;
+  color: #34404b;
```
[the remark this answers is not in the document]
Comment (ManoloAlvarezForo) [reply src/components/EasyPill/EasyPill.module.css:9]: Makes sense I will do it

File: src/components/EasyPill/EasyPill.module.css [hunk 2]
```diff
@@ -2,3 +2,11 @@
   align-self: center;
   margin-right: var(--rvr-space-md);
 }
+
+.tooltipText {
+  height: 24px;
+  line-height: 24px;
```
[the remark this answers is not in the document]
Comment (ManoloAlvarezForo) [reply src/components/EasyPill/EasyPill.module.css:8]: OK, I will change it.

File: src/components/EasyPill/EasyPill.tsx [hunk 3]
```diff
@@ -92,7 +112,7 @@ export const EasyPill: React.FC<EasyPillProps> = ({
                 onDelete(e);
               }}
             >
-              <Icon style={{ display: 'block' }} name="clear" />
+              {renderIfTooltip()}
```
[the remark this answers is not in the document]
Comment (ManoloAlvarezForo) [reply src/components/EasyPill/EasyPill.tsx:115]: OK I will change it.

**Comment by ManoloAlvarezForo** [comment]
Thanks @alexanderAlatorreGoveaCision @ftrinidad-cision @dkordik for your code review, I resolved all observations

**Comment by dkordik** [comment]
sorry, I shoulda mentioned this earlier- can you post screenshots for this kinda stuff?

File: src/components/EasyPill/EasyPill.module.css [hunk 4]
```diff
@@ -2,3 +2,11 @@
   align-self: center;
   margin-right: var(--rvr-space-md);
 }
+
+.tooltipText {
+  height: var(--rvr-line-height-lg);
```
Comment (dkordik) [inline src/components/EasyPill/EasyPill.module.css:7]: is the tooltip height the same as the line height? would the element height not just flow with the content?

Comment (ManoloAlvarezForo) [reply src/components/EasyPill/EasyPill.module.css:7]: Yeah, that's right, I will change it.

**Review by dkordik — APPROVED** [review]

**Pull request merged**